### PR TITLE
Change output format

### LINKED ISSUE #261: Change output format from dolos analysis
The current output format puts all the fragment data into the `intersections.csv` file, while this is fine for smaller files this can get quite big for more realistic examples. Putting all the fragment data into the csv file has a considerable effect on the web UI as it is forced to load in all the fragments in all at once. Putting the this data into separate files would help resolve these issues.

### PR DESCRIPTION
**Pull request by ISteampowered**
closes #261 
depends on #255

### CODE REVIEW

File: web/src/api/api.ts [hunk 1]
```diff
@@ -170,6 +179,11 @@ function parseMetadata(data: d3.DSVRowArray): Metadata {
   );
 }
 
+export async function fetchBlocks(diffId: number, kmers: ObjMap<Kmer>): Promise<Array<Block>> {
```
Comment (rien) [inline web/src/api/api.ts:182]: See my comment in another PR. I would:
- Keep the `blocks` field in `Diff`, but make it nullable.
- Use this method to populate a `Diff` with its blocks.

File: cli/src/lib/presenter/webPresenter.ts [hunk 2]
```diff
@@ -23,6 +23,11 @@ export class WebPresenter extends CsvPresenter {
     app.get("/data/files.csv", respondwithCSV(o => this.writeFiles(o)));
     app.get("/data/diffs.csv", respondwithCSV(o => this.writeDiffs(o)));
     app.get("/data/kmers.csv", respondwithCSV(o => this.writeKmers(o)));
+    app.get("/data/blocks/:id.json", ((req, res) => {
+      res.contentType("text/json");
```
Comment (rien) [inline cli/src/lib/presenter/webPresenter.ts:27]: 404 handling.

**Review by rien — CHANGES_REQUESTED** [review]
I've mistakenly put comments for this PR somewhere else. In short:
- Keep `blocks` in `Diff` interface and populate them with `populateBlocks`.
- Use a `Map` in `WebPresenter` to fetch diffs by id. Handle 404.

**Comment by ISteampowered** [comment]
These changes are included in #262

**Pull request closed without merging**